src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/features/database/update/UpdateDatabaseDataContext.kt

```diff
@@ -3,28 +3,31 @@ package me.seclerp.rider.plugins.efcore.features.database.update
 import com.intellij.openapi.project.Project
 import me.seclerp.observables.bind
 import me.seclerp.observables.observable
+import me.seclerp.observables.observableList
 import me.seclerp.rider.plugins.efcore.features.shared.ObservableMigrations
 import me.seclerp.rider.plugins.efcore.features.shared.dialog.CommonDataContext
 import me.seclerp.rider.plugins.efcore.state.DialogsStateService
 
 class UpdateDatabaseDataContext(intellijProject: Project): CommonDataContext(intellijProject, true) {
-    val availableMigrations = ObservableMigrations(intellijProject, migrationsProject, dbContext)
+    val observableMigrations = ObservableMigrations(intellijProject, migrationsProject, dbContext)
 
-    var targetMigration = observable("")
+    val availableMigrationNames = observableList<String>()
+        .apply {
+            bind(observableMigrations) { migrations -> migrations
+                .map { it.migrationLongName }
+                .sortedByDescending { it }
+            }
+            add("0")
+        }
```

Comment:
Could you try do something like that? Sorry, missed that part before, in current code it will add 0 only 
```suggestion
        .apply {
            bind(observableMigrations) { migrations -> migrations
                .map { it.migrationLongName }
                .sortedByDescending { it }
                .apply { add("0") }
            }
        }
```